bciers/apps/reporting/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -1,7 +1,5 @@
 import NextAuth, { NextAuthOptions } from 'next-auth';
-import { authOptions as regAuthOptions } from 'registration/api/auth/[...nextauth]/route';
-
-export const authOptions = regAuthOptions as NextAuthOptions;
```

Comment:
Nx enforces the Next conventions and verifies that we're not exporting extra stuff from our routes files